chatter.js

```diff
@@ -1,7 +1,8 @@
 Chatter = {
   options: {
     messageLimit: 30,
-    nickProperty: "username",
+    roomLimit: 15,
```

Comment:
I guess same thinking can be applied to messages, like whatsapp, if you scroll far back and want to see older messages you can load more messages